controls/srg_ctr/SRG-APP-000109-CTR-000215.yml

```diff
@@ -5,21 +5,7 @@ controls:
   title: {{{ full_name }}} must take appropriate action upon an audit failure.
   rules:
   - audit_error_alert_exists
-  status: does not meet
-  status_justification: |-
-    OpenShift's architecture is not structured to allow it to specifically
-    halt on a failure to log.  The same database and API that power
-    recording events are shared for normal access, so interactivity with the
-    cluster would be lost in the event of a logging component failure, but
-    workloads would continue operating.  This is by design, as the system
-    is architected to attempt to recover from failure of components. In
-    many failure modes, automated remediations would likely take place in
-    the event of a logging failure.  The system will continue to operate
-    during the remediation attempts.
-  mitigation: |-
-    The OpenShift Platform will generate an alert upon failure of the
-    audit logging service. The responses to those alerts may be defined
-    by the organizations administrative group, and may even be automated
-    responses.  If there is a failure to forward logs to the organization's
-    log collection service, OpenShift will retain about 10 logs up to
-    100MB each, more than that, the logs will then rotate.
+  - auditd_log_format
```

Comment:
I think yes, this SRG is mentioned twice in the STIG, once for the row identified with CCI-000134  and then an extra rule (CCI-000140) for audit alerts.
That said, I think if we run into issues with spreadsheet generation (and I think we will), it would be OK to just include the rule for audit alerts and put the other rules into `related_rules`. I'm fine doing that now or in a subsequent patch that would deal with the spreadsheet generation, just tell me what you prefer.

Reply:
> I think yes, this SRG is mentioned twice in the STIG, once for the row identified with CCI-000134 and then an extra rule (CCI-000140) for audit alerts. That said, I think if we run into issues with spreadsheet generation (and I think we will), 

Hmm, when reviewing I wasn't paying much attention to the spreadsheet generation aspect, but mostly whether the rules make sense to the assigned SRGs, and the consistency of what is checked and remediated.

> it would be OK to just include the rule for audit alerts and put the other rules into `related_rules`. I'm fine doing that now or in a subsequent patch that would deal with the spreadsheet generation, just tell me what you prefer.

Let's have this done now.

Reply:
OK, great, let me try working on that (maybe I'll add some spreadsheet metadata to the controls we have already merged first to see how everything works, the AU family patch is by far the biggest and most involved..)